Library/Formula/sngrep.rb

```diff
@@ -0,0 +1,29 @@
+require 'open3'
+
+class Sngrep < Formula
+  homepage "https://github.com/irontec/sngrep"
+  url "https://github.com/irontec/sngrep/archive/v0.2.2.tar.gz"
+  sha1 "c73a6bbc92f2d6a729566af5179fd550a8f74dbd"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "openssl"
+
+  def install
+    system "./bootstrap.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--with-openssl=#{Formula["openssl"].opt_prefix}"
+    system "make", "install"
+  end
+
+  test do
+    Open3.popen2("#{bin}/sngrep", "-I", test_fixtures("test.pcap")) {|stdin, stdout, wait_thr|
+      stdin.puts "Q"
+      stdin.close
+      wait_thr.value == 0
+    }
```

Comment:
This isn't asserting anything. `pipe_output`, `shell_output` or `system` are probably better calls than `popen2`